content/en/security/cloud_siem/detect_and_monitor/custom_detection_rules/sequence.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Sequence
+description: Learn about how the sequence detection method works.
+disable_toc: false
+---
+
+## Overview
+
+The sequence method enables you to detect multi-stage attacks by identifying ordered patterns of related events, such as initial access, privilege escalation, and data exfiltration.
+
+You can define a sequence of steps that must occur within a defined time frame and across related entities, such as a user, host, or IP address. Each sequence can combine conditions from multiple logs or signals to identify coordinated activity that might be missed by individual rules.
+
+{{< img src="security/security_monitoring/detection_rules/sequence/preview.png" alt="Sequence editor page showing a preview of the steps" style="width:100%;" >}}
+
+See [Create Rule][1] for instructions on how to configure a content anomaly rule.
+
+## How the sequence method works
+
+### Detection logic
+
+{{< img src="security/security_monitoring/detection_rules/sequence/steps.png" alt="Sequence editor page showing three steps" style="width:100%;" >}}
+
+Sequence detection evaluates a defined series of steps that represent distinct stages of suspicious behavior. Each step corresponds to:
+
+- A condition such as a threshold on a log query or a signal match
+- Transitions that define the order and time constraints between steps
+
+When all steps occur in the specified order and within the configured time windows, a signal is generated.
```

Comment:
"A signal is generated" seems abstract and passive. Looking around at other security pages, it looks consistent with how we describe rules, so no pressing need to change it, but maybe something a little more active:
```suggestion
The rule is triggered when all steps occur in the specified order and within the configured time windows.
```